src/middlewared/middlewared/alert/service/any_webhook.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+from middlewared.alert.base import ThreadedAlertService
+from middlewared.schema import Dict, Str
+from middlewared.utils.network import INTERNET_TIMEOUT
+
+class AnyWebhookAlerteService(ThreadedAlertService):
+    title = "AnyWebhook"
+
+    schema = Dict(
+        "AnyWebhook_attributes",
+        Str("url", required=True, empty=False),
+        Str("message_key", requierd=True, empty=False)
+    )
+
+    def send_sync(self, alerts, gone_alerts, new_alerts):
+        r = requests.get(
```

Comment:
`get` method semantics does not suit well such kinds of requests, doesn't it? What kind of alert receiving software are you using?

Reply:
Mostly I think about my GSM provider that allow some SMS sending through an webhook. 
I prefer purpose an open solution instead of an "vendor lock" solution.